### PR TITLE
add last step image configuration

### PR DESCRIPTION
**Pull request by franzholz**
I always forget the real last step.

### CODE REVIEW

File: Documentation/QuickInstall/TheInstallTool/Index.rst [hunk 1]
```diff
@@ -76,6 +76,13 @@ The Admin Tool
       Proceed with :ref:`install-introduction-package`
 
 
+#. In the last step you verify the image handling settings under the configuration presets
```
Comment (mbrodala) [inline Documentation/QuickInstall/TheInstallTool/Index.rst:79]: That's not the last step for setup however. It's a check which can be done independently from the setup.

File: Documentation/QuickInstall/TheInstallTool/Index.rst [hunk 2]
```diff
@@ -76,6 +76,13 @@ The Admin Tool
       Proceed with :ref:`install-introduction-package`
 
 
+#. In the last step you verify the image handling settings under the configuration presets 
+   for Image Magick and Graphics Magick.
+   E.g in Windows you must enter without any quotes ::
+
+     C:\Program Files\ImageMagick-7.0.11-Q8
```
Comment (mbrodala) [inline Documentation/QuickInstall/TheInstallTool/Index.rst:83]: This part does not really make sense. What is the user supposed to then? If this should be a hint that IM/GM should be installed and this path should be set in TYPO3 then we should say so. However I assume this is already spelled out clearly by TYPO3 itself ...

**Review by mbrodala — CHANGES_REQUESTED** [review]

File: Documentation/QuickInstall/TheInstallTool/Index.rst (hunk 1 above)
Comment (mbrodala) [reply Documentation/QuickInstall/TheInstallTool/Index.rst:79]: That's true but still this is not part of the installation setup process. We should mention this separately after the official steps.

File: Documentation/QuickInstall/TheInstallTool/Index.rst (hunk 2 above)
Comment (mbrodala) [reply Documentation/QuickInstall/TheInstallTool/Index.rst:83]: You should at least mention (using `:guilabel:` and/or `$GLOBALS['TYPO3_CONF_VARS']` path) where this path should be entered.

File: Documentation/QuickInstall/TheInstallTool/Index.rst [hunk 3]
```diff
@@ -76,6 +76,21 @@ The Admin Tool
       Proceed with :ref:`install-introduction-package`
 
 
+
+After the last step of the installation you must verify the image handling settings
```
Comment (susannemoog) [inline Documentation/QuickInstall/TheInstallTool/Index.rst:80]: IMHO this is the wrong location for this documentation. The next section (after the one changed here) is still on the last page of the installer. There is no way to access the config for image settings from the installer, without simultaneously completing the installer. 
I would argue that the image settings belong to https://docs.typo3.org/m/typo3/guide-installation/10.4/en-us/In-depth/TheInstallTool/Index.html#the-install-tool-in-depth (at the bottom, image handling).

**Review by susannemoog — CHANGES_REQUESTED** [review]

**Comment by linawolf** [comment]
I agree with Susy. Please move this to  https://docs.typo3.org/m/typo3/guide-installation/10.4/en-us/In-depth/TheInstallTool/Index.html#the-install-tool-in-depth and make a new PR about that. I am closing this PR for now.

**Pull request closed without merging**